default.nix

```diff
@@ -0,0 +1,13 @@
+{
+  # Profiles
+  profiles-common = import ./profiles/common.nix;
+  profiles-server = import ./profiles/server.nix;
+  profiles-desktop = import ./profiles/desktop.nix;
```

Comment:
I think these are actually our main modules while the below behave more like mixins that you can add on top.

Reply:
Something that indicates that you should add one of these and than some of the below would be nice.

Reply:
On the other hand already the categories profiles and roles on its own are confusing enough..

Reply:
Good point. I moved things around a bit.